### ISSUE TITLE
how use BCE loss train single_label data? 

### ISSUE THREAD

**Issue by NeekHua** [open-mmlab/mmclassification#714]
I read cross entropy loss code,find BCE loss used for multitask cls. so model config set this is error .loss=dict(type='CrossEntropyLoss',use_sigmoid=True,loss_weight=1.0),
because pred dim isn't same as label dim.
If I want use BCE loss train bin cls task,how should I write model config for loss?
LooK forward your reply.thks.

**Comment by NeekHua** [comment]
I rewrite mmcls/mdoels/losses/cross_entropy_loss.py binary_cross_entropy functions
add two lines:
````
pred=pred.squeeze()
label=label.float()
````
and my mode config is :
````
# model settings
model = dict(
    type='ImageClassifier',
    backbone=dict(type='MobileNetV2', widen_factor=0.25),
    neck=dict(type='GlobalAveragePooling'),
    head=dict(
        type='LinearClsHead',
        num_classes=1,
        in_channels=1280,
        loss=dict(type='CrossEntropyLoss',use_sigmoid=True,loss_weight=1.0),
        topk=(1,),
        cal_acc=True))
````
It's ok?
any reply for me is so helpful!!!

**Comment by Ezra-Yu** [comment]
Can you show us your output pred size and the label size?

**Comment by NeekHua** [comment]
pred size (batch,1)  label size(batch,1) @Ezra-Yu

**Comment by NeekHua** [comment]
I find through this method,train loss don't decline.
I want to know that mmclassification BCE loss whether used train Single label dichotomy?

**Comment by tianwen0110** [comment]
can you solve this problem? I meet a same one
